src/components/EventPage.js

```diff
@@ -3,38 +3,76 @@ import React, { Component } from "react";
 import { StyleSheet, View } from "react-native";
 
 // Vendor
-import { Button, Text } from "native-base";
+import {
+  Container, Header, Content, H1, H2, H3, Button, Text,
+} from 'native-base';
 
 // Redadalertas
 import { colors } from "styles";
 
 const styles = StyleSheet.create({
-  container: {
+  view: {
     flex: 1,
+    flexDirection: "row",
     justifyContent: "center",
-    alignItems: "center",
+    alignItems: "stretch",
     backgroundColor: "#F5FCFF",
     color: "#000000"
+  },
+  content: {
+    padding: 20
   }
 });
 
+const types = [
+  { label: "Raid", value: "sweep" },
+  { label: "Individual", value: "targeted" },
+  { label: "Traffic", value: "traffic" },
+  { label: "I-9 Audit", value: "i9" },
+  { label: "Checkpoint", value: "checkpoint" },
+  { label: "Action", value: "action" },
+  { label: "False Alarm", value: "falsealarm" },
+  { label: "Other", value: "other" },
+];
```

Comment:
Not crucial right now, but it would be good to consolidate these types between here, the report form, and event map.

Reply:
Same! Hoping we actually get Redux implemented soon.